lefthook.yml

```
@@ -0,0 +1,67 @@
+        fi
+
+post-commit:
+  commands:
+    update-claude:
+      run: |
+        # Update CLAUDE.md with recent commits if needed
+        echo "✅ Commit successful!"
+
+# Install hooks on npm/pnpm install
+scripts:
+  "postinstall": "lefthook install"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)